LEAF_Nexus/api/RESTfulResponse.php

```diff
@@ -79,9 +87,14 @@ public function handler($action)
     /**
      * Outputs in specified format based on $_GET['format']
      * Default to JSON
-     * @param string $out
+     *
+     * @param string|array $out
+     *
+     * @return void
+     *
+     * Created at: 12/2/2022, 1:27:24 PM (America/New_York)
      */
-    public function output($out = '')
+    public function output(string|array $out = ''): void
```

Comment:
I didn't notice this initially, but this actually seems to sometimes get null.  This causes an issue when editing/saving a position title (not sure where else).